File: Models/Entities/Shop/Product.cs
using System.Collections.Generic;

namespace Models.Entities.Shop
{
	public class Product: BaseEntity
	{
		public decimal Price { get; set; }  
        public string ImageUrl {get; set;}
		public string Title {get ;set;}
		public string Description {get; set;}
        public Category Category {get; set;}

		public virtual ICollection<OrderProducts> OrderProducts {get; set;}
	}
}

File: Server/ClientApp/src/app/shop/catalog/models/product.ts
export interface Product {
	id: string;
	title: string;
	price: number;
	category: string;
	imageUrl: string;
	description: string;
}

export function mockProduct(): Product {
	return {
		id: '1',
		title: 'Clean code',
		price: 35.5,
		category: "1",
		imageUrl: "image.png",
		description: 'clean code is a ...'
	}
}